pkg/cmd/cli/backuplocation/create.go

```diff
@@ -86,6 +89,7 @@ func NewCreateOptions() *CreateOptions {
 func (o *CreateOptions) BindFlags(flags *pflag.FlagSet) {
 	flags.StringVar(&o.Provider, "provider", o.Provider, "Name of the backup storage provider (e.g. aws, azure, gcp).")
 	flags.StringVar(&o.Bucket, "bucket", o.Bucket, "Name of the object storage bucket where backups should be stored.")
+	flags.Var(&o.Credential, "credential", "The one credential to be used by this location in key-value pair, where key is the secret name, and value is the secret key name. Optional.")
```

Comment:
I think this could be expanded a little to be more clear.

`The credential to be used by this location as a key-value pair, where the key is the Kubernetes Secret name, and the value is the data key name within the Secret. Optional, one value only.`

I like specifying the `Kubernetes Secret` versus the data key inside of it, to be clear what we're referring to. I'm not too tied to exact verbiage, other than we distinctly identify the key/value here, since they're easy to confuse.